tests/test_transfer.py

```
@@ -720,3 +720,293 @@ def test_transfer_non_overlapping_actions_no_validation(self) -> None:
+        )
+        template = CmabBernoulli.cold_start(
+            action_ids={_ACTION_ID}, n_features=new_features, hidden_dim_list=new_hidden, strategy=ClassicBandit()
+        )
+        result = edit_model_on_the_fly(current, template)
+
+        current_state = json.loads(current.get_state()[1])
+        result_state = json.loads(result.get_state()[1])
+        current_layers = current_state["actions_manager"]["actions"][_ACTION_ID]["model_params"]["bnn_layer_params"]
+        result_layers = result_state["actions_manager"]["actions"][_ACTION_ID]["model_params"]["bnn_layer_params"]
+
+        for layer_idx, (c_layer, r_layer) in enumerate(zip(current_layers, result_layers)):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)